public/js/app/components/CourseFacts.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react'
+import i18n from '../../../../i18n'
 
 import { linkToSchool } from '../util/links'
+import { getDateFormat, seasonStr } from '../util/helpers'
 
-const formatRounds = rounds => {
+const formatRoundsShort = (language, memoData) => {
   // Split rounds with comma after end parentheses and then add '),' in display
-  const splitRounds = rounds.split('),')
-  const lastIndex = splitRounds.length - 1
+  const langIndex = language === 'en' ? 0 : 1
+  const splitRounds = memoData.memoName.split('),')
```

Comment:
Better to use variable destructure in the beginning, it is best practice and has better readability than `memoData.something`, example of destructure, `const {memoName, semester, applicationCodes} = memoData`

Reply:
done